src/data-browser/graphs/SectionSelector.jsx

```diff
@@ -1,26 +1,43 @@
-import './SectionSelector.css'
-import { useCallback } from 'react'
-
+import "./SectionSelector.css"
+import { Link } from "react-router-dom"
+import { useSelector } from "react-redux"
+import { useCallback } from "react"
+import { GRAPH_URL } from "./slice/graphConfigs"
+import { graphs } from "../graphs/slice"
 
 const SectionOption = ({ isSelected, title, onChange }) => {
-  const sectionClasses = `section ${isSelected && 'selected'}`
-  const handleClick = useCallback(_event => onChange(title), [onChange])
+  const sectionClasses = `section ${isSelected && "selected"}`
+
+  const graphStore = useSelector(({ graphs }) => graphs)
+  const graphURL = graphs.getConfig(graphStore, GRAPH_URL) // Getting graph url string from redux store
+  const handleClick = useCallback((_event) => onChange(title), [onChange])
   let ariaLabel = `Navigate to the ${title} tab.`
-  if (isSelected) ariaLabel += ' This section is currently selected.'
+  if (isSelected) ariaLabel += " This section is currently selected."
 
   return (
-    <button
-      className={sectionClasses}
-      aria-label={ariaLabel}
-      onClick={handleClick}
+    // Generating links to direct user to '/graphs' (original graph they were viewing), '/filer' and '/faq'
+    <Link
+      to={
+        title === "Graphs"
+          ? graphURL
+          : `/data-browser/graphs/quarterly/info/${title
+              .replace(" ", "-")
+              .toLowerCase()}`
+      }
     >
-      {title}
-    </button>
+      <button
```

Comment:
Does this still need to be a <button>?  Seems like it's interaction is no longer required since the <Link /> element now handles the redirection, instead of relying on the Component state.
 
 I think the <NavLink /> from `react-router` also accepts `className`, so the styling might also be easily transferable.

Reply:
The `<Link />` component from `react-router` is responsible for handling the URL being updated with the respective tab.

If I remove the `<Link />` the URL no longer gets updated upon a tab selection. 

I tried removed `className` from the `<button />` to the `<Link />` component and it yielded interesting results.

<img width="252" alt="image" src="https://user-images.githubusercontent.com/60801873/190448147-4b09f26a-11d5-43bf-9263-5406cc24a507.png">

Reply:
Sorry, I didn't realize my element tags had been removed from the comment.
 
 I was asking if the `children` of the \<Link /> component needed to be a \<button>, since the \<Link /> is now handling the navigation. 
 
Also it looks like react-router's \<NavLink /> might be able to better apply the className?  Also review the styles in [SectionSelector.css](https://github.com/cfpb/hmda-frontend/blob/master/src/data-browser/graphs/SectionSelector.css) to see if you can adjust them to apply to the new anchor elements.

Reply:
Resolved.